uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Validator/DateTest.php

```diff
@@ -34,7 +34,11 @@ public function testCanValidateDateInDifferentFormats($date, $format)
   public function datesDataProvider()
   {
     return [
-      [date('Y-m-d'), true],
+      [CRM_Utils_Date::processDate(date('Y-m-d')), true],
+      [CRM_Utils_Date::processDate(date('Y-m')), true],
+      [CRM_Utils_Date::processDate(date('Y')), true],
+      [CRM_Utils_Date::processDate(date('m')), true],
+      [date('Y-m-d'), false],
```

Comment:
Wouldn't it be better to just use date('Ymd') instead of process date? Also, I'm not sure if all those dates should be valid